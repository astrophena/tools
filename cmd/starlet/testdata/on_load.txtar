update: forwarded
-- bot.star --
def send_message(to, text, reply_markup = {}):
    return telegram.call(
        method = "sendMessage",
        args = {
            "chat_id": to,
            "text": text,
            "reply_markup": reply_markup
        }
    )

def on_load():
    send_message(config.owner_id, "Loaded!")

def handle(update):
    send_message(config.owner_id, "Hello!")
