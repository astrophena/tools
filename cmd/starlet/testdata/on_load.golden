[
  {
    "method": "sendMessage",
    "args": {
      "chat_id": 123456789,
      "reply_markup": {},
      "text": "Loaded!"
    }
  },
  {
    "method": "sendMessage",
    "args": {
      "chat_id": 123456789,
      "reply_markup": {},
      "text": "Hello!"
    }
  }
]